### PR TITLE
feat: krisp plugin integration

### PR DESCRIPTION
**Pull request by raviteja83**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        gap: '$4',
+        alignItems: 'center',
+      }}
+    >
+      <Flex css={{ alignItems: 'center', justifyContent: 'center', '& svg': { size: '$8' } }}>{icon}</Flex> {children}
+    </Dropdown.Label>
+  );
+};
+
+const plugin = new HMSKrispPlugin();
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx:90]: Should we initialise in a separate file like VB?

File: packages/hms-virtual-background/.gitignore [hunk 2]
```diff
@@ -1,4 +0,0 @@
-*.log
-.DS_Store
-node_modules
-dist
```
Comment (KaustubhKumar05) [inline packages/hms-virtual-background/.gitignore:4]: Why was this file changed?

File: packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+              bg: '$surface_default',
+            }}
+          >
+            {children}
+          </Dropdown.Content>
+        </Dropdown.Root>
+      ) : null}
+    </Flex>
+  );
+};
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx:151]: image

Some change in IconButtonWithOptions or AVToggle has affected this

File: packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+import { ToastManager } from './Toast/ToastManager';
+import { Dropdown } from '../../Dropdown';
+import { Label } from '../../Label';
+import { Flex } from '../../Layout';
+import { Switch } from '../../Switch';
+import { Tooltip } from '../../Tooltip';
+import IconButton from '../IconButton';
+import { isMacOS } from '../common/constants';
+
+// const optionsCSS = { fontWeight: '$semiBold', color: '$on_surface_high', w: '100%' };
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx:37]: image

Is this the final design? It's a lot to take in. We're not showing for mweb?

**Comment by KaustubhKumar05** [comment]
https://www.loom.com/share/0f3ac28ebfa648e09bf7ba32e6b114d8
The state is not being perserved

File: packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx (hunk 3 above)
Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx:151]: Borders should be rounded when there is no options section on the tigh

File: packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx (hunk 4 above)
Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx:37]: mweb will need to check once.

File: packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx (hunk 3 above)
Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx:151]: when I checked in mweb it seemed fine. let me check

File: packages/hms-virtual-background/.gitignore (hunk 2 above)
Comment (raviteja83) [reply packages/hms-virtual-background/.gitignore:4]: not sure, don't remember making the change

Comment (raviteja83) [reply packages/hms-virtual-background/.gitignore:4]: I guess it is not needed at package level. root level .gitignore handles

File: packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx (hunk 3 above)
Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/IconButtonWithOptions/IconButtonWithOptions.tsx:151]: This is on desktop when the mic permissions are disabled

File: packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx (hunk 1 above)
Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/AudioVideoToggle.tsx:90]: since it is only used here and only 1 line added here

**Review by KaustubhKumar05 — APPROVED** [review]

**Review by KaustubhKumar05 — APPROVED** [review]
